src/components/Results.vue

```diff
@@ -1,71 +1,79 @@
 <template>
-  <div v-if="q && data && data">
+  <div v-if="q && data">
     <div
       v-for="dataset in data.terms"
       :key="dataset.source.uri"
       class="container"
     >
       <h2 class="display-5 text-center py-3">
         {{ dataset.source.name }}
-        <small class="termCount">({{ t('search.termsFound', dataset.terms.length) }})</small>
+        <small v-if="dataset.source.alternateName">({{ dataset.source.alternateName }})</small>
+        <small class="termCount">: {{ t('search.termsFound', dataset.terms.length) }}</small>
+        <p>
+          <small class="termCount">{{ dataset.source.creators[0].name }}</small>
+        </p>
       </h2>
-      <div class="card-columns">
+      <div>
         <div
           v-for="term in dataset.terms"
           :key="term.uri"
-          class="card shadow-sm"
+          class="card shadow-sm mb-3"
```

Comment:
Style consideration: the cards are less dense now (because of their full width). This makes the result listing as a whole look at bit 'pale': the background color of both the page and the cards is white. Would it be an idea to give the cards a different background color, for contrast (e.g. `#F8F9FA`)?

Reply:
Done.